### ISSUE TITLE
Auto save media being sent

### ISSUE THREAD

**Issue by JetUni** [klinker-apps/pulse-sms-issues#974]
## Confirm this Issue

- [x] I have [searched](https://github.com/klinker-apps/messenger-issues/issues), by keyword, for my request to verify that there aren't any open or closed issues similar to this behavior.
- [x] I have [searched the help page](https://messenger.klinkerapps.com/help) to verify that there are not any topics discussing similar behavior.
- [x] This is not a general support questions. Support should be done over [email](mailto:luke@klinkerapps.com).

## Describe Your Request

I'd like to be able to have the app save all the media that I send as well as receive. Right now it only saves what I receive. There are times when I take a picture through the app and it's one that I would like to be saved, but I don't always remember to go back and click on the image and save it to the device. It would be nice if sent messages were part of the automatic saving feature that you already have.

### Describe alternatives you've considered

The alternative at the moment is that I have to manually save the images each time I take them.

### Additional information

This is something that another app, Textra, does that I really miss after moving from it to Pulse

**Comment by klinker24** [comment]
But what about the times when you are sending something that you didn't take in the app? It doesn't really make sense to save them again. That will just duplicate the image and the duplicate images will appear throughout your phone.

Honestly, I don't think that this is a feature that I will code or officially support in the app. If you wanted to add it yourself, there is an open source contributions section of the settings page that you could add it into. I discuss that on the app's readme

https://github.com/klinker-apps/pulse-sms-android

**Comment by JetUni** [comment]
Good point, however, the point that they would be saved is after they are taken with the camera. On the app I work on at my job, there is an option to save the images that are taken. This would limit it so that you wouldn't have duplicate images showing up when images are sent that are already in the gallery.

**Comment by klinker24** [comment]
Feel free to implement this, if you choose. I can add it to my backlog for the future, but can't say when I will be able to get to it.
